hack/install-ocs-operator.sh

```diff
@@ -28,6 +28,7 @@ data:
   CSI_ENABLE_TOPOLOGY: "test"
   CSI_TOPOLOGY_DOMAIN_LABELS: "test"
   ROOK_CSI_ENABLE_NFS: "false"
+  CSI_DISABLE_HOLDER_PODS: "true"
```

Comment:
@parth-gr can you please update the PR description on how this will solve the problem? i assume this file is only for dev testing not used in any released deployment.

Reply:
Correct added the changes